pkg/tc/driver/cmdline/tc_cmdline_test.go

```diff
@@ -0,0 +1,415 @@
+package driver_test
+
+import (
+	"github.com/Mellanox/multi-networkpolicy-tc/pkg/policyrules"
```

Comment:
import order

Reply:
i really need to check how to sort it in ide